src/components/ReportActionItem/TaskPreview.js

```diff
@@ -54,7 +55,7 @@ function TaskPreview(props) {
     // The reportAction might not contain details regarding the taskReport
     // Only the direct parent reportAction will contain details about the taskReport
     // Other linked reportActions will only contain the taskReportID and we will grab the details from there
-    const isTaskCompleted = props.taskReport
+    const isTaskCompleted = !_.isEmpty(props.taskReport)
```

Comment:
I should have caught in the proposal stage but we're deprecating underscore soon (we aim to use only lodash)

furthermore, we move toward to use pure JS when possible.

so in this case, we can do
```suggestion
    const isTaskCompleted = props.taskReport !== {}
```

Reply:
Ah sorry I did see that proposal to deprecate underscore, but didn't see the resolution! `props.taskReport !== {}` will always return true, so I've gone with the suggestion from that Slack thread of `Object.keys(obj).length !== 0`.